src/crypto/PSASpake2p.h

```diff
@@ -144,14 +145,14 @@ class PSASpake2p_P256_SHA256_HKDF_HMAC
     CHIP_ERROR KeyConfirm(const uint8_t * in, size_t in_len);
 
     /**
-     * @brief Return the shared secret.
+     * @brief Return the shared HKDF key.
```

Comment:
Maybe `Return the HKDF Key handle containing a key reference to the shared secret`?

Reply:
Ok, I just copied the docs from `CHIPCryptoPALPSA::GetKeys`, but extending it can be valuable.